e2etests/run_athens.go

```diff
@@ -0,0 +1,56 @@
+// +build e2etests
+
+package e2etests
+
+import (
+	"fmt"
+	"net/http"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"time"
+)
+
+func buildAthens(goBin string, destPath string, env []string) (string, error) {
+	target := path.Join(destPath, "athens-proxy")
+	binFolder, err := filepath.Abs("../cmd/proxy")
+	if err != nil {
+		return "", fmt.Errorf("Failed to get athens source path %v", err)
+	}
+
+	cmd := exec.Command(goBin, "build", "-o", target, binFolder)
+	cmd.Env = env
+	err = cmd.Run()
+	return target, err
+}
+
+func stopAthens() error {
+	cmd := exec.Command("pkill", "athens-proxy")
+	err := cmd.Run()
+	return err
+}
+
+func runAthensAndWait(athensBin string, env []string) error {
+	cmd := exec.Command(athensBin)
+	cmd.Env = env
+
+	go func() {
```

Comment:
why not just `go cmd.Run()` ?

Reply:
and if you are trying to run the command in the background, it is more appropriate to use cmd.Start(): https://golang.org/pkg/os/exec/#Cmd.Start

Reply:
> and if you are trying to run the command in the background, it is more appropriate to use cmd.Start(): https://golang.org/pkg/os/exec/#Cmd.Start

Ah, nice!